calico/getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -88,7 +88,7 @@ Before you get started, make sure you have downloaded and configured the {% incl
 1. Now that you have an empty cluster configured, you can install the Tigera operator. 
 
    ```bash
-   kubectl apply -f {{site.data.versions.first.manifests_url}}/manifests/tigera-operator.yaml
+   kubectl create -f {{site.data.versions.first.manifests_url}}/manifests/tigera-operator.yaml
    ```
 
```

Comment:
Perhaps make this change more obvious to people - its easy to gloss over the change.
```suggestion
   > **Note**: Due to the large size of the CRD bundle, `kubectl apply` might exceed request limits. Instead, use `kubectl create` or `kubectl replace`.
   {: .alert .alert-info}

```

Reply:
Yes, good point. Thanks. However, I think it would suffice to have this note just at the top instead of all the places.